src/angular-wysiwyg.js

```diff
@@ -337,6 +342,15 @@ Requires:
                 scope.setHiliteColor = function() {
                     scope.format('hiliteColor', scope.hiliteColor)
                 }
+                
+                scope.setCssClass= function() {
+                    var classes = scope.cssClasses.join(' ');
+                    var selection = window.getSelection().focusNode.parentNode;
+                    $(selection).removeClass(classes);
```

Comment:
Instead of using jQuery can this be accomplished with angular's jqlite (angular.element() )?